### PR TITLE
fixed footers and button up position

### PR DESCRIPTION
**Pull request by mscibisz**
This change does not include v4.8.0.3 yet.
It is to fix the position of the footer and button 'up'. It does take into consideration the possibility of having uBlock or AddBlock active.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1123,16 +1123,36 @@ <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.0.
...
+                        document.getElementById('responsive-footer').style.bottom = "48px";
+                        document.getElementById('up-button').style.bottom = "3.5rem";
+                    }
+                }
+
+                var x = window.matchMedia("(max-width: 980px)")
+                myFunction(x) // Call listener function at run time
+                x.addListener(myFunction) // Attach listener function on state changes
+      } else {
+        console.log("Thank you for not using AddBlock!");
```
Comment (miczyg1) [inline index.html]: `AddBlock`? @mscibisz

Comment (mscibisz) [reply index.html]: AdBlock* :)

Comment (miczyg1) [reply index.html]: @mscibisz is this gonna be fixed or it should be like that?

Comment (mscibisz) [reply index.html]: Oh, I thought I just made that typo only in the comment above! Yeah, fixing now.

**Pull request merged**